packages/block-editor/src/components/child-layout-control/index.js

```diff
@@ -223,7 +223,11 @@ export default function ChildLayoutControl( {
 									} }
 									value={ columnStart }
 									min={ 1 }
-									max={ parentLayout?.columnCount }
+									max={
+										parentLayout?.columnCount -
+										columnSpan +
```

Comment:
I'm seeing a `Warning: Received NaN for the `max` attribute.` which I think is coming from here; `columnCount` doesn't always exist so this might be NaN on auto layouts.

Reply:
Still seeing this error here on Auto layouts.

Reply:
Correction: I also see it in Manual mode when clicking "Pin to grid" without moving the block.

Reply:
Fixed. (I think.)